SW.SecurityService.Core/Models/AuthenticationAnswer.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace SW.SecurityService.Core.Models
+{
+    public class AuthenticationAnswer
+    {
+        public UserRedis userRedis { get; set; }
+
+        public bool wrongPassword { get; set; } = false;
+
+        public bool nonExistenLogin { get; set; } = false;
```

Comment:
We should Capitalize `NonExistenLogin`.
Also we need to consider using is prefix for bool properties. Like:
`IsWrongPassword`
`IsNonExistingUser`